build.gradle

```diff
@@ -34,4 +34,8 @@ dependencies {
     testCompile 'org.mockito:mockito-core:1.9.5'
     testCompile "com.squareup.okhttp3:mockwebserver:${okhttp_version}"
     testCompile 'org.assertj:assertj-core:3.4.1'
-}
\ No newline at end of file
+    compile "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
+}
+sourceSets {
+    main.java.srcDirs += 'src/main/kotlin'
```

Comment:
I was at the same point - but it is really better to do all kotlin files into the java directory and do not separate as it would be good for OCD - would have to search the project/commit but I have been at the same place and gone back as this can cause problems with the tooling ..